### ISSUE TITLE
pyodbc reads random characters in column name when description is set for this column

### ISSUE THREAD

**Issue by denfromufa** [mkleehammer/pyodbc#124]
```
def win_connect_mdb(mdb_path):
    if sys.platform not in ('win32','cli'):
        raise Exception('This function is available for use in Windows only.')
    mdb_driver = [d for d in pyodbc.drivers() if 'Microsoft Access Driver (*.mdb' in d]
    if mdb_driver == []:
        raise Exception('Access Driver is not found.')
    else:
        driver_name = mdb_driver[0]
    return pyodbc.connect('Driver={'+driver_name+"};DBQ="+mdb_path, readonly = False, ansi = True)

connection1 = win_connect_mdb(path1)
cur1=connection1.cursor()
pd.DataFrame([list(_) for _ in cur1.columns(table1)])
```

**Comment by mkleehammer** [comment]
I've rebased the v4 branch so I could make a smaller pull request #160.

**Comment by mkleehammer** [comment]
I believe Unicode issues are fixed in the newly released 4.0.0 version.
